sdk/eventhub/event-hubs/src/batchingReceiver.ts

```diff
@@ -163,6 +193,9 @@ export class BatchingReceiver extends EventHubReceiver {
         receiver.removeListener(ReceiverEvents.receiverError, onReceiveError);
         receiver.removeListener(ReceiverEvents.message, onReceiveMessage);
         receiver.session.removeListener(SessionEvents.sessionError, onReceiveError);
+        if (aborter) {
+          aborter.removeEventListener("abort", onAbort);
+        }
```

Comment:
In `detached()` we should remove the listener if we decide not to re-open the link